octobot_trading/exchanges/config/ccxt_exchange_settings.py

```diff
@@ -0,0 +1,199 @@
+from octobot_commons import enums as common_enums
+from octobot_trading import enums as enums
+from octobot_trading.exchanges.util import parser
+import octobot_trading.exchanges.util.exchange_market_status_fixer as exchange_market_status_fixer
+
+
+class CCXTExchangeConfig:
+    """
+    override this class if you need custom settings
+    """
+
+    # available methods to choose from
+    ALL_GET_ORDER_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_ORDER_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_PRIVATE.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_OPEN_AND_CLOSED_ORDERS.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_USING_STOP_ID.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_TRADES.value,
+    ]
+    ALL_GET_ALL_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_ALL_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_ALL_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    ALL_GET_OPEN_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_OPEN_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_OPEN_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    ALL_GET_CLOSED_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_CLOSED_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_CLOSED_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    ALL_CANCEL_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.CANCEL_ORDER_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.CANCEL_STOP_ORDER_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    ALL_GET_MY_RECENT_TRADES_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_USING_RECENT_TRADES.value,
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_USING_CLOSED_ORDERS.value,
+    ]
+    ALL_GET_POSITION_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_POSITION_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_BY_SUB_TYPE.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_WITH_PRIVATE_GET_POSITION_RISK.value,
+    ]
+
+    # set default exchange config here
+
+    # overwrite classes if you need a different parser
+    ORDERS_PARSER_CLASS = parser.OrdersParser
+    TRADES_PARSER_CLASS = parser.TradesParser
+    POSITIONS_PARSER_CLASS = parser.PositionsParser
+    MARKET_STATUS_FIXER = exchange_market_status_fixer.ExchangeMarketStatusFixer
+    
+    USE_FIXED_MARKET_STATUS = None  # None -> default is with fixed market status (but without precision fixing)
+    #                                 False -> define False if not necessary
+    #                                 True -> set to True if exchange is know to need fixed market status (with precision fixing)
```

Comment:
Maybe an enum here could be better. We could also include the "remove limits" case in this enum

Reply:
I planed to move all related stuff into a market status parser class. And automate erything except precision and limits so there will be those two booleans in the end